docs/advisories/cve_2017_14491.md

```diff
@@ -0,0 +1,129 @@
+# CVE-2017-14491
+
+A vulnerability in dnsmasq, used by kube-dns, requires an upgrade to the kube-dns component. This component is the default DNS component installed in Kubernetes.  The vulnerability may be externally exploitable. Links below exist with the full detail
+of the CVE. This is not a Kubernetes specific vunerability, but exists in dnsmasq.
+
+## Current kops Status
+
+As of 2017/10/08 `kops` fixes are not in a released `kops` version, but you are
+able to hotfix any Kubernetes cluster that is 1.4.x or higher.
+
+## Tested Kubernetes Versions
+
+Kubernetes versions 1.5.8, 1.6.11, 1.7.7, and 1.8.0 have been fully tested with the new version of `kube-dns` deployment.  Other versions should function, but upgrading
+to fully tested version is recommend. We have had 1.4.x users upgrade successfully,
+but we cannot validate full production stability.  Local testing in a non-production
+environment is always recommended. We are not able to quatify the risk of using
+a non-tested version.
+
+## Fixed kops releases
+
+We are planning releasing fixes in both new 1.7.x and 1.8.x kops releases.
+Fixes are in master, but not yet released as of 2017/10/08. We will update this document when releases are available. You are able to continue to use kops, and
+apply the hotfix.  kops should not downgrade the hotfix.
+
+### Fixed kops Version Matrix
+
+| kops Version  | Fixed  | Released | Will Fix | URL |
```

Comment:
Something went wrong with the formatting of this

Reply:
Missed a column. Let me make a couple more tweaks with the color anyways

Reply:
Fixed